client/app/models/milestone.js

```diff
@@ -152,8 +153,16 @@ export default class MilestoneModel extends Model {
       displayDate = this.dcpActualstartdate;
     }
 
-    if (this.dcpMilestone === FILED_EAS_REVIEW) {
-      displayDate = this.dcpActualstartdate;
+    if (this.dcpMilestone === FILED_EAS_REVIEW) { // FILED_EAS_REVIEW is used here as the generic "EAS" milestone
+      // 2792: if the project starts with P, prefare the display date found in the "PREPARE_FIELD_EAS" milestone
+      if (this.project.get('dcpName').startsWith('P')) {
+        const prepareFiledEas = this.project.get('milestones').findBy('dcpMilestone', PREPARE_FILED_EAS);
+        if (prepareFiledEas) {
+          displayDate = prepareFiledEas.dcpActualenddate;
+        }
+      } else {
+        displayDate = this.dcpActualstartdate;
+      }
```

Comment:
Amaazzing!!!!!! ⭐ 🌟  ⭐ 🌟  ⭐ 🌟  so clear!

Reply:
So FILED_EAS_REVIEW is the same as "Review Filed EAS" in the ticket?

Reply:
> So FILED_EAS_REVIEW is the same as "Review Filed EAS" in the ticket?

Yep! I could clean up the names sooner or later...